pkg/telemetry/telemetry.go

```diff
@@ -151,7 +150,7 @@ func (c Component) sendTelemetry(ctx context.Context) {
 
 	c.log.WithField("data", data).WithField("hostdata", hostData).Info("sending telemetry")
 	if err := c.analyticsClient.Enqueue(analytics.Track{
-		AnonymousId: machineID(),
+		AnonymousId: "(removed)",
```

Comment:
AnonymousID is the Segment entity: https://segment.com/docs/connections/sources/catalog/libraries/website/javascript/identity/#anonymous-ids

As far as I can see it's not used in our data pipeline now, but are we sure, that we want to remove it completely?

Reply:
I would think the unique cluster id is more important for telemetry than the machine id 🤔

Reply:
Yeah, of course, I was wondering if it could break the pipeline, but we checked with Chen and it should be ok. 
Chen suggested testing it a bit, but in general we are fine